sdk/core/amqp-common/src/requestResponseLink.ts

```diff
@@ -122,25 +123,61 @@ export class RequestResponseLink implements ReqResLink {
           request.message_id = generate_uuid();
         }
 
+        const rejectOnAbort = () => {
+          const address = this.receiver.address || "address";
+          const requestName = options!.requestName || "requestName";
+          const desc: string =
+            `[${this.connection.id}] The request "${requestName}" ` +
+            `to "${address}" has been cancelled by the user.`;
+          log.error(desc);
+          const error = translate(
+            new Error(`The request operation has been cancelled by the user.`)
```

Comment:
Use the `requestName` variable here
`The ${requestName} operation has been cancelled by the user.`